community/concepts/deep-then-wide.md

```diff
@@ -1,30 +1,51 @@
 # deep then wide
 
-SourceCred is still a very young project. As we grow it, we have a choice of priorities: 
+SourceCred is still a young project. As we grow it, we have a choice of priorities:
 
- - do we want to go "wide" (adoption by lots of projects) or 
- - "deep" (intensive usage by a few projects)?
-
-If you are familiar with depth vs. breadth when it comes to searching, this idea
-is similar to that. In the next sections, we'll talk about each option, and
-what it would mean for development and the future of the project.
+ - do we want to go "wide" (adoption by lots of projects) or
+ - go "deep" (intensive usage by a few projects)?
 
 #### What does it mean to go wide?
 
-Going "wide" is easier. SourceCred can run on any GitHub repository, so we have a path for engagement with the entire GitHub community. We could focus on getting more and more projects to try running SourceCred, and then convert them to regular usage. If we went down this route, we would focus on features that make the existing cred scores more salient and engaging — for example, making it easy to generate "weekly reports" showing the new cred being earned by contributors.
+Going "wide" means encouraging lots of communities to start using SourceCred.
+Since SourceCred can run on any GitHub repository or Discourse forum, we could
+roll cred out to many different projects and groups. We could optimize for the
+number of projects using SourceCred, and focus on features that make the scores
+more salient or engaging — for example, making it easy to generate "weekly
+reports" showing the new cred being earned by contributors.
 
-As an added bonus to going wide, progress is much easier to measure, as it’s easier to quantify the number of projects using SourceCred, but hard to quantify the amount of value that SourceCred is adding to any particular project.
+Going wide is tempting because progress is very easy to measure, and the project
+would get plenty of buzz which could bring more contributors and opportunities.
```

Comment:
"plenty of buzz which" -> "plenty of buzz that"

Typically with a "which" you expect a comma, e.g., ""plenty of buzz, which"

And I would suggest changing "bring" to "attract" or something to suggest drawing people to the project. I would either remove opportunities or expand on it - what does it mean when someone brings an opportunity? What is an example? E.g., you could just say:

> Going wide is tempting because progress is very easy to measure, and the project
would get plenty of buzz that could attract more contributors.

The entire discussion about "what is an opportunity for sourcecred, as a project" is probably not appropriate to draw out here, imho (but maybe somewhere else?)

Reply:
Changed which to that. I think "bring" is fine (and "attracting" opportunities reads a little strange to me).